### ISSUE TITLE
Wrong dependency to @angular/cdk@^9.0.0-rc.8

### ISSUE THREAD

**Issue by MicMicMon** [angular/flex-layout#1195]
# Bug Report

After upgrading to Angular v9, I get this on `yarn install`:

```sh
warning " > @angular/flex-layout@9.0.0-beta.29" has incorrect peer dependency "@angular/cdk@^9.0.0-rc.8"
```

**Comment by fabiannagel** [comment]
I got the same warning after trying to update @angular/material:

`Package "@angular/flex-layout" has an incompatible peer dependency to "@angular/cdk" (requires "^8.0.0-rc.0", would install "9.0.0").`

**Comment by MicMicMon** [comment]
This is different, you need to put `9.0.0-beta.29` as flex-layout version in your `packages.json`.

**Comment by CaerusKaru** [comment]
This looks like an issue with the Angular CLI, not Flex Layout. Our peer dep is still valid (it resolves to 9.0.0, which is correct).
